.storybook/routes.js

```diff
@@ -41,6 +41,7 @@ const routes = {
   '/components/text/subtext/': 'components-text-subtext--basic',
   '/components/text/text/': 'components-text-text--basic',
   '/components/text/title/': 'components-text-title--basic',
+  '/examples/layout/': 'examples-layouts--area-column-positioning',
```

Comment:
`/examples/layout` doesn't exist on the site yet, but I'm adding it here for when it's eventually added in the future.